### PR TITLE
BBS updated dataset

### PR DESCRIPTION
**Pull request by Kaian**
Updated Dataset with new API split (platform, brand, client) and business endpoints.

### CODE REVIEW

File: library/Ivoz/Provider/Domain/Model/DestinationRateGroup/DestinationRateGroupDto.php [hunk 1]
```diff
@@ -6,6 +6,26 @@ class DestinationRateGroupDto extends DestinationRateGroupDtoAbstract
...
+     * @inheritdoc
+     */
+    public static function getPropertyMap(string $context = '')
+    {
+        if ($context === self::CONTEXT_COLLECTION) {
+            return [
+                'status' => 'status',
+                'id' => 'id',
+                'name' => ['en', 'es'],
+                'description' => ['en', 'es'],
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/DestinationRateGroup/DestinationRateGroupDto.php]: I would not show description in collection requests

File: library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php [hunk 2]
```diff
@@ -4,5 +4,21 @@
...
+    /**
+     * @inheritdoc
+     */
+    public static function getPropertyMap(string $context = '')
+    {
+        if ($context === self::CONTEXT_COLLECTION) {
+            return [
+                'id' => 'id',
+                'name' => ['en','es'],
+                'description' => ['en','es'],
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/RatingPlanGroup/RatingPlanGroupDto.php]: I would not show description in collection requests

File: web/rest/brand/features/provider/currency/getCurrency.feature [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+              "id": 3,
+              "name": {
+                  "en": "Pound",
+                  "es": "Libra"
+              }
+          }
+      ]
+    """
+
+  Scenario: Retrieve certain country json
```
Comment (mmadariaga) [inline web/rest/brand/features/provider/currency/getCurrency.feature]: ~~country~~

**Comment by Kaian** [comment]
Updated, I'll mark the PR ready for review :D

**Comment by Kaian** [comment]
I forgot to update the test also :sweat_smile:

**Pull request merged**